brands/amenity/bureau_de_change.json

```diff
@@ -61,6 +61,7 @@
   },
   "amenity/bureau_de_change|Беларусбанк": {
     "locationSet": {"include": ["001"]},
```

Comment:
Can you add the specific location for this?

Reply:
Ok just added this.